ngx-tools/src/set-form-control-value/set-form-control-value.ts

```diff
@@ -0,0 +1,17 @@
+import { FormGroup } from '@angular/forms';
+
+
+export function setFormControlValue(form: FormGroup, controlName: string, controlValue: string): any {
+  if (!form || !controlName) {
+    return false;
+  }
+
+  const control = form.get(controlName);
+
+  if (control) {
+    control.setValue(controlValue);
+    return true;
+  } else {
+    return false;
```

Comment:
Is there a use-case where the return value is used by the consumer?

Reply:
Not currently. But if not returning boolean, we're gonna return nothing?

Reply:
If there's no value in what is returned, I don't really see a reason to.

On line 6 you can just `return;` to stop the flow. Beyond that, all we care about is setting the form control.

This will give us slightly better tests too as it will force us to call the function then check the actual value of the formControl rather than just checking the return of this function.